Directory.Packages.props

```diff
@@ -15,13 +15,15 @@
     <PackageVersion Include="Microsoft.SqlServer.Server" Version="1.0.0" />
     <PackageVersion Include="Microsoft.SqlServer.TransactSql.ScriptDom" Version="$(ScriptDomPackageVersion)" />
     <PackageVersion Include="Microsoft.SqlServer.Types" Version="160.1000.6" />
+    <PackageVersion Include="Nuget.Versioning" Version="6.13.2" />
```

Comment:
still need this framework?

Reply:
We need it for parsing version and comparison, with support of prerelease versions. Alternatively we can use [Semver](https://www.nuget.org/packages/semver/) but I thought Nuget one is better since Microsoft owns it.